config/application.rb

```diff
@@ -51,6 +51,11 @@ class Application < Rails::Application
     # Defaults to false in production and test, true otherwise.
     config.sequel.schema_dump = false
 
+    # This setting disables the automatic connect after Rails init. It prevents
+    # rake assets:precompile from triggering a database connection during the
+    # Docker image build.
+    config.sequel.skip_connect = true
```

Comment:
This needs to be on by default with an exception if we are precompiling assets otherwise we won't know there is a problem with the database connection until the first client auth attempt which will lead to all kinds of UX problems I think.

Reply:
I changed to
```
    config.sequel.skip_connect = true if is_asset_precompile?
```

Reply:
Btw, the errors seem unconnected to this issue and probably a bigger problem than it might seem:  
```
...
As you have more than one active sequel database you have to specify the one to use manually!
```
You may want to look into this separately as to why just running the app connects to sequel twice.

Reply:
Aha - this thing is probably doing it for you already: https://github.com/cyberark/conjur/pull/1390/files?file-filters%5B%5D=.dev&file-filters%5B%5D=.feature&file-filters%5B%5D=.rb&file-filters%5B%5D=.yml&file-filters%5B%5D=No+extension#diff-e16a910aa68736855918fb08ee37f808R19